cl_dll/hudgl.cpp

```diff
@@ -6,7 +6,9 @@
 #include <vector>
 
 #ifdef _WIN32
+#include <winsani_in.h>
```

Comment:
What's this for?

Reply:
These headers are needed so that `ARRAYSIZE` and `HSPRITE` do not conflict with the HLSDK code, because Microsoft has its own implementation for them

Even if these changes are not directly related to this request, I see no reason not to add them, and I don’t want to create a separate request for that kind of stuff